packages/grid/x-data-grid-pro/src/tests/editComponents.DataGridPro.old.test.tsx

```diff
@@ -22,6 +22,11 @@ import { spy } from 'sinon';
 
 const isJSDOM = /jsdom/.test(window.navigator.userAgent);
 
+function fireDoubleClickEvent(cell: HTMLElement) {
+  fireEvent.mouseUp(cell);
```

Comment:
For consistency:

```suggestion
  fireEvent.mouseDown(cell);
```

or

```suggestion
  fireEvent.mouseUp(cell);
  fireEvent.mouseDown(cell);
```